pkg/cloud/azure/services/resources/template/deployment-params.json

```
@@ -33,7 +33,7 @@
         "value": "Standard_B2ms"
     },
     "location": {
-        "value": "eastus"
+        "value": "eastus2"
```

I am really a `westus2` person :P
```suggestion
        "value": "westus2"
```

---

Heh. This file is actually from the older implementations of capz. 
We never copy this into the `cluster-api-azure-controller`, so I'll delete it.